x-pack/test/security_solution_api_integration/config/services/security_solution_endpoint_data_stream_helper.ts

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
Move the shared utils to service to avoid type check confusion:

<img width="2486" alt="Screenshot 2024-07-03 at 11 47 32" src="https://github.com/elastic/kibana/assets/6295984/67f7d075-1eea-40b6-b237-cc17f1ee71fc">